src/map/battle.c

```diff
@@ -4661,13 +4661,17 @@ static struct Damage battle_calc_weapon_attack(struct block_list *src, struct bl
 	{
 		short cri = sstatus->cri;
 		if (sd != NULL) {
+			// Racial crit boni are affected by katar's crit bonus.
```

Comment:
```suggestion
			// Racial crit bonus are affected by katar's crit bonus.
```

Reply:
Im willing to settle for bonuses as english is unfortunately a joke of a language that does'nt use the only true plural "boni" of bonus

Reply:
Applied "bonuses"